website/docs/Pipelines/Exome_Germline_Single_Sample_Pipeline/README.md

```diff
@@ -108,6 +108,8 @@ Additionally, the 4 GQ band schema has specific improvements compared with the 7
 2. It has no overlaps; the only overlapping positions should be two variants (i.e. deletions) on separate haplotypes.
 3. No more no-calls; all genotypes should be called. Positions with no data will be homRef with GQ0.
 
+Read more about the reblocked GVCFs in the [WARP Blog](https://broadinstitute.github.io/warp/blog/Blog%201).
```

Comment:
just a nit/curiosity but why is the URL Blog%201 ? wouldn't it have something more specific about the title of the blog post?

Reply:
Not nit... I changed to Nov21_ReblockedGVCF. This was my first time playing with the blog functionality- I was more focused on getting the blog to work/be formatted correctly then looking at the URL, which was pretty ugly. Thanks for calling out.